acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/GithubCreationPage.java

```diff
@@ -130,10 +130,7 @@ public void completeCreationFlow(String apiKey, String org, String pipeline, boo
         }
         selectOrganization(org);
 
-        wait.until(singlePipelineBtn,120000)
```

Comment:
there is no wait now - and there is an intermittent failure on create so this may actually have made the test less stable (it might be trying to do something before it gets a chance to load the screen)